web/src/components/ErrorScreen.js

```diff
@@ -1,13 +1,11 @@
 import React from 'react';
 import Guild from './Guild';
 import {UserAvatar} from './DiscordIcon';
+import {CSSTransitionGroup} from 'react-transition-group';
 
-export default class ErrorScreen extends React.Component {
-  render() {
-    const props = this.makeInner();
-    if (!props) return null;
-
-    return (
+const ErrorScreen = props => {

```

Comment:
an actual `function` just to keep consistent with all the other functional components we have

also, you can destructure props in the args

(we've done this already; see `CurrentSong`)

Reply:
🐇